test/logout.tests.js

```diff
@@ -44,7 +44,7 @@ describe('logout route', function() {
 
     it('should redirect to the base url', function() {
       assert.equal(logoutResponse.statusCode, 302);
-      assert.equal(logoutResponse.headers.location, 'https://example.org');
+      assert.equal(logoutResponse.headers.location, 'https://example.org/');
```

Comment:
🤔 why did this change?

Reply:
Either that, or

```diff
- postLogoutRedirectUri: Joi.string().uri({allowRelative: true}).optional().default('/')
+ postLogoutRedirectUri: Joi.string().uri({allowRelative: true}).optional().default('')
```

The default `'/'` gets appended to the baseURL.

Omitting `.default()` crashes the tests with:
```js
UnhandledPromiseRejectionWarning: TypeError [ERR_INVALID_ARG_TYPE]: The "url" argument must be of type string. Received type undefined
```